builder/azure/arm/config.go

```diff
@@ -151,6 +151,9 @@ type Config struct {
 
 	Comm communicator.Config `mapstructure:",squash"`
 	ctx  *interpolate.Context
+
+	//Cleanup
+	AsyncRGDelete bool `mapstructure:"async_resourcegroup_delete"`
```

Comment:
Other config names expand RG to ResourceGroup, so please be consistent.

Reply:
fixed